dev/src/write-batch.ts

```diff
@@ -589,7 +573,7 @@ export class WriteBatch {
     return this._firestore
       .request<api.ICommitRequest, api.CommitResponse>('commit', request, tag)
       .then(resp => {
-        const writeResults: WriteResult[] = [];
+        let writeResults: WriteResult[] = [];
 
         if (request.writes!.length > 0) {
           assert(
```

Comment:
This assert is not needed anymore - we no longer use index-based lookups in this map. That means that you can remove the assert and the intermediate writeResult variable.